pulp_docker/app/models.py

```diff
@@ -179,37 +158,22 @@ class ManifestTag(Content):
 
     name = models.CharField(max_length=255, db_index=True)
 
-    manifest = models.ForeignKey(
-        ImageManifest, null=True, related_name='manifest_tags', on_delete=models.CASCADE)
-
-    class Meta:
-        unique_together = (
-            ('name', 'manifest'),
-        )
-
-
-class ManifestListTag(Content):
-    """
-    A tagged Manifest List.
+    tagged_manifest = models.ForeignKey(
+        Manifest, null=True, related_name='tagged_manifests', on_delete=models.CASCADE)
 
-    Fields:
-        name (models.CharField): The tag name.
-
-    Relations:
-        manifest_list (models.ForeignKey): A referenced Manifest List.
-
-    """
-
-    TYPE = 'manifest-list-tag'
-
-    name = models.CharField(max_length=255, db_index=True)
-
-    manifest_list = models.ForeignKey(
-        ManifestList, null=True, related_name='manifest_list_tags', on_delete=models.CASCADE)
+    media_type = models.CharField(
```

Comment:
what is the reason of having the media_type field on the tag model? we can retrieve from the manifest it is references

Reply:
removed